packages/odyssey-react-mui/src/Banner.tsx

```diff
@@ -10,11 +10,19 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import { AlertColor, AlertProps } from "@mui/material";
+import { AlertProps } from "@mui/material";
 import { memo } from "react";
 import { Alert, AlertTitle, Link, ScreenReaderText } from "./";
 import { useTranslation } from "react-i18next";
 
+export const bannerRoleValues = ["status", "alert"] as const;
+export const bannerSeverityValues = [
+  "success",
+  "info",
+  "warning",
+  "error",
+] as const;
```

Comment:
Do we want this to be:
```suggestion
export const bannerSeverityValues: AlertColor[] = [
  "success",
  "info",
  "warning",
  "error",
] as const;
```